docs/platforms/javascript/common/configuration/integrations/genericpool.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Mongoose
+description: "Adds instrumentation for Generic Pool. (default)"
+supported:
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.fastify
+  - javascript.gcp-functions
+  - javascript.hapi
+  - javascript.koa
+  - javascript.nestjs
+  - javascript.electron
+  - javascript.nextjs
+  - javascript.nuxt
+  - javascript.sveltekit
+  - javascript.remix
+  - javascript.astro
+  - javascript.bun
+---
+
+<Alert level="info">
+
+This integration only works in the Node.js and Bun runtimes. Requires SDK version `8.29.0` or higher.
+
+</Alert>
+
+_Import name: `Sentry.genericPoolIntegration`_
+
+This integration is enabled by default when performance monitoring is enabled. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).
+
+The `genericPoolIntegration` adds instrumentation for the `generic-pool` library to capture spans using [`@opentelemetry/instrumentation-generic-pool`](https://www.npmjs.com/package/@opentelemetry/instrumentation-generic-pool).
+
+## Supported Versions
+
+- `mongoose`: `>=2.0.0 <4`
```

Comment:
Just want to double check that this should be mongoose

Reply:
it should not! sorry for that.